src/compiler/types.ts

```diff
@@ -989,7 +990,7 @@ namespace ts {
         _unaryExpressionBrand: any;
     }
 
-    export interface IncrementExpression extends UnaryExpression {
+    export interface UpdateExpression extends UnaryExpression {

```

Comment:
This is presumably a breaking change? Might want to update https://github.com/Microsoft/TypeScript/wiki/Breaking-Changes